### PR TITLE
WIP: main/llvm-friends: bump to 8.0.0

### PR DESCRIPTION
**Pull request by Cogitri**
This depends on LLVM8 being moved to main/

### CODE REVIEW

**Comment by J0WI** [comment]
There's also `community/llvm-libunwind`

**Comment by Cogitri** [comment]
Oh right, missed that one. Thanks!

File: community/lld/APKBUILD [hunk 1]
```diff
@@ -45,11 +45,12 @@ check() {
 		armhf|armv7) rm ../test/ELF/eh-frame-hdr-augmentation.s;;
 	esac
 
-	#make check-lld
+	# make check-lld doesn't like using system lit
```
Comment (J0WI) [inline community/lld/APKBUILD]: I was trying with `LLVM_EXTERNAL_LIT` in #8533

Comment (Cogitri) [reply community/lld/APKBUILD]: I think calling lit manually should do the same thing, but thanks for the pointer!

**Pull request closed without merging**